model/src/pyrenew/process/simplerandomwalk.py

```diff
@@ -76,11 +86,12 @@ def transition(x_prev, _):
             xs=jnp.arange(n_timepoints - 1),
         )
 
-        return (jnp.hstack([init, x]),)
+        return (jnp.hstack([init, x.flatten()]),)
```

Comment:
Did the previous version lead to any problems?

Reply:
Yes, `x` was ending up of shape `(1, n_timepoints)` rather than shape `(n_timepoints,)`

Reply:
Can we write a test based on this?

Reply:
What did you have in mind here?

Reply:
Something about the change from 

```python
diff = npro.sample(name + "_diffs", self.error_distribution)
```

to 

```
diff, *_ = self.step_rv.sample(**kwargs)
```
Unclear why the latter has a different shape though. This also seems relevant to the question of non-tuple RV return types.

Reply:
Ah, I wasn't accounting for that change. There was no error in the version before this PR?

Reply:
Not that I saw, anyway.

Reply:
Then I'm fine without a test.